src/safeds/data/image/containers/_image.py

```diff
@@ -23,71 +30,106 @@ class Image:
 
     Parameters
     ----------
-    data : BinaryIO
-        The image data as bytes.
+    image_tensor : Tensor
+        The image data as tensor.
     """
 
+    _pil_to_tensor = PILToTensor()
+    _default_device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
```

Comment:
The device is not only relevant for images but also for NNs. Thus, this should be moved to another location. I'd suggest something like the following in `src/safeds/config/_device.py` (+ and entry in the corresponding `__init__.py`):

```py
def _get_device() -> ?:
    return torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
```

We can later think about if, when, and how, the device can be overridden.

Reply:
see #524